From 8a9bfe37750af6bbdc7ab71d44ffcbb978b7f61f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mika=20Bostr=C3=B6m?= <mika.bostrom@nomovok.com>
Date: Thu, 19 Apr 2012 19:34:24 +0300
Subject: [PATCH 3/4] Install public headers

Add public headers as install target. This allows to build things that
depend on Qt3D and Qt3DQuick components.
---
 src/quick3d/quick3d.pro |    4 ++++
 src/threed/threed.pro   |    5 +++++
 2 files changed, 9 insertions(+), 0 deletions(-)

diff --git a/src/quick3d/quick3d.pro b/src/quick3d/quick3d.pro
index f4026e6..d0ab505 100644
--- a/src/quick3d/quick3d.pro
+++ b/src/quick3d/quick3d.pro
@@ -37,6 +37,10 @@ DEFINES += QT_BUILD_QT3D_QUICK_LIB
 
 !contains(QT_CONFIG, egl):DEFINES += QT_NO_EGL
 
+devel.files = $$PUBLIC_HEADERS
+devel.path = $$[QT_INSTALL_HEADERS]/$$TARGET
+
+INSTALLS += devel
 
 
 
diff --git a/src/threed/threed.pro b/src/threed/threed.pro
index 7f82777..b527dd9 100644
--- a/src/threed/threed.pro
+++ b/src/threed/threed.pro
@@ -25,3 +25,8 @@ HEADERS += $$PRIVATE_HEADERS
 DEFINES += QT_BUILD_QT3D_LIB
 
 !contains(QT_CONFIG, egl):DEFINES += QT_NO_EGL
+
+devel.files = $$PUBLIC_HEADERS
+devel.path = $$[QT_INSTALL_HEADERS]/$$TARGET
+
+INSTALLS += devel
-- 
1.7.5.4

